composer.json

```diff
@@ -11,16 +11,16 @@
     ],
     "require": {
         "php": ">=5.4",
-        "symfony/symfony": "~2.6",
+        "symfony/symfony": "~3",
         "symfony/monolog-bundle": "~2.4",
         "doctrine/orm": "~2.5",
         "doctrine/doctrine-bundle": "~1.5",
         "sensio/framework-extra-bundle": "~3.0",
 
-        "gedmo/doctrine-extensions": "dev-master",
+        "gedmo/doctrine-extensions": "2.4.*",
         "stfalcon/tinymce-bundle": "~1.1|~2.1",
 
-        "nfq-rho/admin-bundle": "~0.5"
+        "nfq-rho/admin-bundle": "dev-sf3_update"
```

Comment:
As I understand, it should be changed when new admin-bundle version will be released.